var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -35,6 +36,8 @@ class Kaldi(Package):    # Does not use Autotools
     depends_on('speex', type='run')
     depends_on('openfst@1.4.1-patch', when='@2015-10-07')
     depends_on('openfst@1.6.0:', when='@2018-07-11')
+    depends_on('openfst@1.6.0:', when='@2019-07-29')
```

Comment:
@jcftang Apologies but I think there will be a similar issue here: versions of `kaldi` after `2019-07-29` will also want to use `openfst@1.6.0:` I assume? I think this should update the `when` to be `@2019-07-29:`. If in fact you anticipate that later versions will not need that constraint, let me know (in which case LGTM).

Reply:
Yea, I wasn't too sure of this one either, but openfst updates have been pretty rare in the kaldi project afaik, I think this is something that should be addressed as new versions of kaldi are added and released. This is a bit different from the cub package which has a higher update cycle.